coursebuilder/modules/explorer/_static/components/pages/feedback-page.html

```diff
@@ -7,166 +7,180 @@
-                        <p>
-                          Still having trouble? Click the General feedback option and let us know.
-                        </p>
+                        Don't worry. There's a good chance your progress is simply linked to one of your other email accounts. Try switching your login details to check any other profiles you might have registered.
+                        <br>
+                        <br>
+                        Still having trouble? Click the General feedback option and let us know.
                         </span></div>
                     </li>
                   </ul>
                 </div>
-                <h3 style="font-size: 1.375em; margin: 1.9em 0 0.9em; color: #4054B2">
+                <h3>
                   Report a bug
                 </h3>
 
                 <div class="row white">
                   <form class="col s12">
                     <div class="row">
                       <div class="input-field col s12">
-                        <input id="email" type="email" class="validate">
+                        <input id="email" type="email" class="validate"
+                               style="border-bottom: 1px solid grey;">
                         <label for="email">Email</label>
                       </div>
                     </div>
                     <div class="row">
                       <div class="input-field col s12">
                         <textarea id="textarea3" class="materialize-textarea"></textarea>
-                        <label for="textarea3">Tell us about content ideas or feedback you'd like to share</label>
+                        <label for="textarea3">Tell us about content ideas or feedback you'd like to
+                          share</label>
                       </div>
-                      <div class="col s12">
-                        0/1000
+                      <div class="col s12"
+                           style="text-align: right; font-size: 12px; color: black;">
+                        0 / 1000
                       </div>
                     </div>
                     <div class="card-action col s12">
-                      <a href="https://freeformers.com/" class="right"
-                         style="color: #4054b2; font-weight: 700; font-family: Roboto,Arial,sans-serif;">SHARE</a>
+                      <a href="https://freeformers.com/" class="right">SHARE</a>
                     </div>
                   </form>
                 </div>
 
-                <div>
-                  <p style="padding: 2rem; line-height: 2.8rem; color: #757575;">
-                    The information you submit will be used in compliance with Google’s <a class="blue-text">Privacy
+                <div class="footer-div">
+                  <p>
+                    The information you submit will be used in compliance with Google’s <a>Privacy
                     Policy</a>. We may contact you at the e-mail address you provided.
```

Comment:
Hmm.. This footer privacy/policy seems to be repeating. Any way to not have it repeating?